rust_src/src/fns.rs

```diff
@@ -255,4 +255,21 @@ pub fn require(feature: LispObject, filename: LispObject, noerror: LispObject) -
 }
 def_lisp_sym!(Qrequire, "require");
 
+/// Concatenate all the arguments and make the result a list.
+/// The result is a list whose elements are the elements of all the arguments.
+/// Each argument may be a list, vector or string.
+/// The last argument is not copied, just used as the tail of the new list.
+/// usage: (append &rest SEQUENCES)
+#[lisp_fn()]
```

Comment:
The build is failing, I think this should be `#[lisp_fn]` without the round parens.